data/json/items/armor/storage.json

```diff
@@ -1260,6 +1260,273 @@
     "flags": [ "BELTED", "WATER_FRIENDLY" ],
     "armor": [ { "encumbrance": [ 5, 20 ], "coverage": 70, "covers": [ "torso" ], "specifically_covers": [ "torso_hanging_back" ] } ]
   },
+  {
+    "id": "molle_assault_pack",
+    "repairs_like": "bigback",
+    "type": "ARMOR",
+    "name": { "str": "MOLLE assault pack" },
+    "description": "The Modular Lightweight Load-carrying Equipment is an advanced military backpack.  Covered with pockets and straps, it strikes a fine balance between storage space and encumbrance.",
+    "weight": "1300 g",
+    "volume": "5000 ml",
+    "price": 6700,
+    "price_postapoc": 1500,
+    "material": [ "nylon", "plastic" ],
+    "symbol": "[",
+    "looks_like": "backpack",
+    "color": "green",
+    "pocket_data": [
+      {
+        "pocket_type": "CONTAINER",
+        "max_contains_volume": "20 L",
+        "max_contains_weight": "35 kg",
+        "max_item_length": "50 cm",
+        "magazine_well": "3 L",
+        "moves": 300
+      }
+    ]
+  },
+  {
+    "id": "molle_assault_pack",
```

Comment:
This item definition is a duplicate of the previous one.

Reply:
I'm cleaning up this PR for them and gonna PR changes to their branch